### DISCUSSION TITLE
Convert directly to android studio  + add python language

### DISCUSSION THREAD

**Discussion by yusufquteb** [Sketchware-Pro/Sketchware-Pro#1007] [Q&A] [answered]
Convert directly to android studio 
Add python language
When you launch the new update for sketchwar pro?

**Comment by j-px** [comment, marked as answer]
1. exists already, see Sign/Export menu
2. you are using an app which is focused on using Blocks and a drag-n-drop view editor, no.
3. this project is too small to have any idea when anything will be finished.
